### PR TITLE
allow more flexibility of changing MB model params

### PR DESCRIPTION
**Pull request by lilianschuster**
- allow to change the prcp_fac and temp_bias within `mb_calibration_from_geodetic_mb` 
-> not sure if this works for the tests and scripts.

- missing in `mb_calibration_from_geodetic_mb` :  docstring for parameters 
- also: maybe remove default returning of calibration parameters?

### CODE REVIEW

**Comment by fmaussion** [comment]
> allow to change the prcp_fac and temp_bias within mb_calibration_from_geodetic_mb

Thanks! Agreed.

> missing in mb_calibration_from_geodetic_mb : docstring for parameters

Yes - definitely on my todo

> also: maybe remove default returning of calibration parameters?

No, I like it - most recent OGGM tasks do have some kind of return. If it annoys you in the notebook, add a `;` at the end of the line (we do the same in many tutorials)

File: oggm/core/massbalance.py [hunk 1]
```diff
@@ -1522,7 +1523,7 @@ def mb_calibration_from_geodetic_mb(gdir,
                                   'you posted!')
 
     # Ok, regardless on how we want to calibrate, we start with defaults
-    temp_bias = 0
+    # temp_bias = 0
```
Comment (fmaussion) [inline oggm/core/massbalance.py]: Can remove this

**Review by fmaussion — APPROVED** [review]
Thanks! What is the empty oggm file in the docs?

**Comment by lilianschuster** [comment]
- ok I removed the strange empty oggm file (I don't know from where it came from)
- another thing that might need to be changed: 
`prcp_fac` is used inside of `MonthlyTIModel`, but `prcp_scaling_factor` inside of `mb_calibration_from_geodetic_mb` and in e.g. `cfg.PARAMS['prcp_scaling_factor_min']`. Maybe it would be good to stick to one naming convention there?

**Comment by fmaussion** [comment]
Yes I agree - I will keep this in mind when I rethink the renaming, but I will do all renaming in one go when things are green.

**Pull request merged**